apps/labs/labs-louisehuyton-nodejs/labs-louisehuyton-nodejs.yaml

```
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: labs-louisehuyton-nodejs
+  namespace: labs
+spec:
+  releaseName: labs-louisehuyton-nodejs
+  chart:
+    spec:
+      chart: ./stable/labs-louisehuyton-nodejs
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    nodejs:
+      image: hmctssandbox.azurecr.io/labs/louisehuyton-nodejs:latest # {"$imagepolicy": "flux-system:labs-louisehuyton-nodejs"}
+      ingressHost: labs-louisehuyton-nodejs-{{ .Values.global.environment }}.platform.hmcts.net
```

this should be 
```suggestion
      ingressHost: labs-louisehuyton-nodejs.{{ .Values.global.environment }}.platform.hmcts.net
```

but it should be inherited from your chart, i.e. it doesn't need to be added here

---

Can see where the '-' is added instead of '.' in the script. Would that need fixed or completely removed from the script due to be being inherited @timja?

---

should be completely removed ingress hosts should be setup in the apps chart

---

cool will do thank you!